<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Broadcaster - multiple pull input stream creation test"
           tests="4"
           failures="1"
           errors="0"
           skipped="3"
           time="0.000">
  <properties />
            <testcase name="testInit" time="0.000" classname="com.zixi.load.testing.BroadcasterMultipePullCreationTest">
                <failure type="java.lang.NoClassDefFoundError"
                  message="com/zixi/load/drivers/BroadcasterMultipleCreationDriver"
              >
<![CDATA[java.lang.NoClassDefFoundError: com/zixi/load/drivers/BroadcasterMultipleCreationDriver
  com.zixi.load.testing.BroadcasterMultipePullCreationTest.testInit(BroadcasterMultipePullCreationTest.java:16)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
  org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
  org.testng.TestRunner.privateRun(TestRunner.java:771)
  org.testng.TestRunner.run(TestRunner.java:621)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1032)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.Surefire.run(Surefire.java:180)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
  org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
Caused by: java.lang.ClassNotFoundException: com.zixi.load.drivers.BroadcasterMultipleCreationDriver
  java.net.URLClassLoader.findClass(URLClassLoader.java:381)
  java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
  java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  com.zixi.load.testing.BroadcasterMultipePullCreationTest.testInit(BroadcasterMultipePullCreationTest.java:16)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:517)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:140)
  org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
  org.testng.TestRunner.privateRun(TestRunner.java:771)
  org.testng.TestRunner.run(TestRunner.java:621)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1032)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.Surefire.run(Surefire.java:180)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
  org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
      </failure>

          </testcase>
  
            <testcase name="afterTest" time="0.0" classname="com.zixi.load.testing.BroadcasterMultipePullCreationTest">
          <skipped />
    </testcase>
            <testcase name="beforeTes" time="0.0" classname="com.zixi.load.testing.BroadcasterMultipePullCreationTest">
          <skipped />
    </testcase>
            <testcase name="broadcasterMultiplePullInCreation" time="0.0" classname="com.zixi.load.testing.BroadcasterMultipePullCreationTest">
          <skipped />
    </testcase>
  
  
</testsuite>
